WeddingSiteBuilder/Views/profileupdate.html

```diff
@@ -118,6 +182,7 @@
         <br>
         First Name: <br /><input type="text" id="FirstName1" maxlength="50"><br />
         Last  Name: <br /><input type="text" id="LastName1" maxlength="50">
+        <input type="hidden" id="AttendeeId1" />
```

Comment:
Check this out, This is how I store variables that we might need later but don't actually need to be displayed